components/navigation.js

```diff
@@ -840,6 +847,9 @@ function NavigationCollapsed({ navigationClick, toggleNavCollapsed }) {
       <NavigationItem>
         <IndexerState isCollapsed />
       </NavigationItem>
+      <NavigationItem>
+        <Email isCollapsed />
```

Comment:
The `Email` component doesn’t have a property named `isCollapsed`. It would normally change the styling for the collapsed menu. Existing ones make the icon bigger for the collapsed menu, thought that might not be needed here. But in any case, you’re passing a property `isCollapsed` to a component that doesn’t accept any properties.

Reply:
I removed `isCollapsed` and rebased the branch to dev.